packages/ethers5/src/client.ts

```diff
@@ -168,8 +171,40 @@ export class Web3Modal extends Web3ModalScaffold {
         WalletConnectProvider.on('display_uri', (uri: string) => {
           onUri(uri)
         })
-
-        await WalletConnectProvider.connect()
+        console.log('SIWEController', SIWEController, siweConfig)
+        console.log('WalletConnectProvider init opts', options)
+
+        if (siweConfig?.options?.enabled) {
+          const result = await WalletConnectProvider.authenticate({
```

Comment:
```suggestion
          const { SIWEController } = await import('@web3modal/siwe')
          
          const result = await WalletConnectProvider.authenticate({
```

Reply:
should we move the `authenticate` function inside `WalletConnectProvider`?

Reply:
I don't understand this point. It's already this way this way - `await WalletConnectProvider.authenticate`

Reply:
you're right nvm lol